socorro/unittest/external/es/test_crashstorage.py

```diff
@@ -132,30 +133,27 @@ def test_invalid_key(self, key):
 class TestIntegrationESCrashStorage(ElasticsearchTestCase):
     """These tests interact with Elasticsearch (or some other external resource)."""
 
-    def setup_method(self, method):
-        super().setup_method(method)
+    def setup_method(self):
+        super().setup_method()
         self.config = self.get_tuned_config(ESCrashStorage)
 
-        # Helpers for interacting with ES outside of the context of a
-        # specific test.
-        self.es_client = elasticsearch.Elasticsearch(
-            hosts=self.config.elasticsearch.elasticsearch_urls
-        )
-        self.index_client = elasticsearch.client.IndicesClient(self.es_client)
-
     def test_index_crash(self):
         """Test indexing a crash document."""
         es_storage = ESCrashStorage(config=self.config)
 
+        raw_crash = deepcopy(SAMPLE_RAW_CRASH)
+        processed_crash = deepcopy(SAMPLE_PROCESSED_CRASH)
+        processed_crash["date_processed"] = date_to_string(utc_now())
+
         es_storage.save_processed_crash(
-            raw_crash=deepcopy(a_raw_crash),
-            processed_crash=deepcopy(a_processed_crash),
+            raw_crash=raw_crash,
+            processed_crash=processed_crash,
         )
 
         # Ensure that the document was indexed by attempting to retreive it.
-        assert self.es_client.get(
-            index=es_storage.es_context.get_index_template(),
```

Comment:
Previously, the template variable didn't include any date placeholders, so you didn't need to interpolate it. That's not true now, so I switched it to use `get_index_for_date()`.